components/build-service/base/build-pipeline-selectors/OWNERS

```diff
@@ -0,0 +1,34 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+
+approvers:
+- mmorhun
+- tkdchen
+- rcerven
+- mkosiarc
+- tisutisu
+- chmeliik
+- dirgim
+- hongweiliu17
+- jsztuka
+- jinqi7
+- sonam1412
+- MartinBasti
+- Josh-Everett
+- 14rcole
```

Comment:
Should we please also add Dheeraj in the list. Github user account: dheerajodha

Reply:
added